### PR TITLE
Missing data for paternity/maternity and earliest articles.

### PR DESCRIPTION
**Pull request by toneillbroad**

### CODE REVIEW

File: src/genegraph/source/graphql/schema/bibliographic_resource.clj [hunk 1]
```diff
@@ -27,5 +32,8 @@
                              :path [:dc/date]}
             :abstract {:type 'String
                        :description "The paper's abstract."
-                       :path [:dc/abstract]}}})
+                       :path [:dc/abstract]}
+            :earliest_article {:type 'Boolean
+                               :description "Article represents the earliest article"
+                               :resolve earliest-article}}})
```
Comment (tnavatar) [inline src/genegraph/source/graphql/schema/bibliographic_resource.clj:38]: I don't think this works--earliest article is a fact derived from the assertion and curation, and not a fact about the article. Would remove this.

File: src/genegraph/source/graphql/schema/proband_evidence.clj [hunk 2]
```diff
@@ -47,4 +47,7 @@
                                 :path [:sepio/detection-method]}
             :ethnicity {:type :Resource
                         :description "The ethnicity of the proband."
-                        :path [:sepio/ethnicity]}}})
+                        :path [:sepio/ethnicity]}
+            :paternity_maternity_confirmed {:type 'String
+                                            :description "Paternity/maternity confirmed."
+                                            :path [:sepio/paternity-maternity-confirmed]}}})
```
Comment (tnavatar) [inline src/genegraph/source/graphql/schema/proband_evidence.clj:53]: What are the possible values of this?

**Review by tnavatar — CHANGES_REQUESTED** [review]

File: src/genegraph/source/graphql/schema/bibliographic_resource.clj (hunk 1 above)
Comment (toneillbroad) [reply src/genegraph/source/graphql/schema/bibliographic_resource.clj:38]: OK. Good reminder not to distribute facts.

File: src/genegraph/source/graphql/schema/proband_evidence.clj (hunk 2 above)
Comment (toneillbroad) [reply src/genegraph/source/graphql/schema/proband_evidence.clj:53]: "Yes", "No", "none" when included.

Comment (tnavatar) [reply src/genegraph/source/graphql/schema/proband_evidence.clj:53]: This feels like some technical debt; but we're under time pressure and it solves the immediate problem. Doing it this way leaves open the possibility of doing it better in the future, so I'm OK with this.

**Review by tnavatar — APPROVED** [review]

**Pull request merged**